### ISSUE TITLE
EMAGE: how to choose which model of VQVAE is used for training EMAGE

### ISSUE THREAD

**Issue by weihaosky** [PantoMatrix/PantoMatrix#144]
In pretrained_vq, the models are of different epochs.
Now I am training VQVAE from scratch.
So after the training of VQVAE, how do I choose which model is used for training EMAGE?
Thanks

**Comment by H-Liu1997** [comment]
we selected the epoch based on best val loss

thanks!

**Comment by alexanderapers** [comment]
Hi @H-Liu1997, I am wondering if training EMAGE with the emage.yaml config file automaticallly uses the pretrained VQ-VAEs from /EMAGE/pretrained_vq? And if so, how can this be changed to using my own version of retrained VQ-VAEs from scratch stored in /outputs/audio2pose/custom.

Also, suppose I want to use the retrained VQ-VAEs that have been trained for 200 epochs, which .bin files correspond to this? last_200.bin, face.bin or other .bin files? 

Thanks!

**Comment by caim33** [comment]
> Hi [@H-Liu1997](https://github.com/H-Liu1997), I am wondering if training EMAGE with the emage.yaml config file automaticallly uses the pretrained VQ-VAEs from /EMAGE/pretrained_vq? And if so, how can this be changed to using my own version of retrained VQ-VAEs from scratch stored in /outputs/audio2pose/custom.
> 
> Also, suppose I want to use the retrained VQ-VAEs that have been trained for 200 epochs, which .bin files correspond to this? last_200.bin, face.bin or other .bin files?
> 
> Thanks!

The same problem, have you solved it? I would be very grateful if you could help me solve this problem
